vendor/init.bat

```diff
@@ -312,8 +313,7 @@ echo :: use in front of the command to prevent printing the command
 echo.
 echo :: the next two lines is for "%%flag_exists%%" shortcut, a custom arguments handler
 echo :: don't remove it if you need it
-echo set "CMDER_USER_FLAGS=%%*"
-echo call "%%cmder_root%%\vendor\lib\flag_exists"
+echo call "%%cmder_root%%\vendor\lib\flag_exists" "/setPath"
```

Comment:
I'm not sure if this `/setPath` is good or the old one(add global varible `%flag_exists%` automatically) is better.